capstone/src/test/java/com/google/sps/clubs/ClubServletTest.java

```diff
@@ -65,70 +69,114 @@ public void setUp() throws IOException {
     Entity clubEntity = datastore.prepare(query).asSingleEntity();
 
-    Assert.assertEquals(SAMPLE_CLUB_NAME, clubEntity.getProperty(Constants.CLUB_NAME_PROP));
+    Assert.assertEquals(SAMPLE_CLUB_NAME, clubEntity.getProperty(Constants.PROPERTY_NAME));
     Assert.assertEquals(SAMPLE_CLUB_DESC_1, clubEntity.getProperty(Constants.DESCRIP_PROP));
     Assert.assertEquals(SAMPLE_CLUB_WEB, clubEntity.getProperty(Constants.WEBSITE_PROP));
     Assert.assertEquals(SAMPLE_BLOB, clubEntity.getProperty(Constants.LOGO_PROP));
     Assert.assertEquals(STUDENT_LIST, clubEntity.getProperty(Constants.MEMBER_PROP));
     Assert.assertEquals(STUDENT_LIST, clubEntity.getProperty(Constants.OFFICER_PROP));
 
-    Mockito.verify(response).sendRedirect(VALID_URL);
+    Mockito.verify(response).sendRedirect("/registration-msg.html?is-valid=true");
   }
 
   @Test
   public void doPost_registerNewInvalidClub() throws ServletException, IOException {
-    helper.setEnvEmail(TEST_EMAIL).setEnvAuthDomain("google.com").setEnvIsLoggedIn(true);
-    when(request.getParameter(Constants.CLUB_NAME_PROP)).thenReturn(SAMPLE_CLUB_NAME);
-    when(request.getParameter(Constants.DESCRIP_PROP)).thenReturn(SAMPLE_CLUB_DESC_1);
-    when(request.getParameter(Constants.WEBSITE_PROP)).thenReturn(SAMPLE_CLUB_WEB);
-
-    ImmutableList<BlobKey> keys = ImmutableList.of(SAMPLE_BLOB);
-    ImmutableMap<String, List<BlobKey>> blobMap = ImmutableMap.of(Constants.LOGO_PROP, keys);
-    when(blobstore.getUploads(request)).thenReturn(blobMap);
-
     clubServlet.doPostHelper(request, response, blobstore, datastore);
 
-    when(request.getParameter(Constants.DESCRIP_PROP)).thenReturn(SAMPLE_CLUB_DESC_2);
+    when(request.getParameter(Constants.DESCRIP_PROP)).thenReturn("club desc");
     clubServlet.doPostHelper(request, response, blobstore, datastore);
 
-    Mockito.verify(response).sendRedirect(INVALID_URL);
+    Mockito.verify(response).sendRedirect("/registration-msg.html?is-valid=false");
     Query query =
         new Query("Club")
             .setFilter(
                 new FilterPredicate(
-                    Constants.CLUB_NAME_PROP,
+                    Constants.PROPERTY_NAME,
                     FilterOperator.EQUAL,
-                    request.getParameter(Constants.CLUB_NAME_PROP)));
+                    request.getParameter(Constants.PROPERTY_NAME)));
     Entity clubEntity = datastore.prepare(query).asSingleEntity();
     Assert.assertEquals(SAMPLE_CLUB_DESC_1, clubEntity.getProperty(Constants.DESCRIP_PROP));
   }
+
+  @Test
+  public void doGet_clubExists() throws ServletException, IOException {
+    ImmutableList<String> expectedMembers =
+        ImmutableList.of("student@example.com", "officer@example.com");
+    ImmutableList<String> expectedOfficers = ImmutableList.of("officer@example.com");
+    ImmutableList<String> expectedAnnouncements = ImmutableList.of("an announcement");
+
+    Entity clubEnt = new Entity("Club");
```

Comment:
nit: I can't find the link right now :( but I read somewhere that it's better to write out the entire word instead of abbreviating so clubEntity instead of clubEnt

Reply:
Found it! https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers - this is for JS though, so up to you if you want to change it or not :)